vm/src/protocol/number.rs

```diff
@@ -238,12 +306,12 @@ impl PyNumber<'_> {
 
     // PyIndex_Check
     pub fn is_index(&self) -> bool {
-        self.methods().index.load().is_some()
+        self.obj.class().slots.number.index.load().is_some()
```

Comment:
binary ops operands can be swapped, but is this also need to access obj.class() again?

Reply:
ah, got it. we didn't handle unary operator yet.